WordPress/twentytwentyfive
# Search block styles in the style variations, option 1

**Description**

This PR changes the border radius of the search block button and search input field.
It also sets the border color of the search input field to accent color 6, the color with the opacity.
In one variation, it also updates the font family for the button.

With this option, the input field styles are limited to this single block, and managed in multiple theme.json files.

**Screenshots**
Noon, after

<img width="708" alt="Search block styles in the Noon variation" src="https://github.com/user-attachments/assets/526accb3-0233-4d9a-acd2-ffd2530f14a6">

Dusk, after:
<img width="708" alt="Search block styles in the Dusk variation" src="https://github.com/user-attachments/assets/82121cfa-e6c7-4b2f-babe-6444bd3aba21">


**Testing Instructions**

Add a search form.
Confirm that the style matches the expectations listed in this issue: https://github.com/WordPress/twentytwentyfive/issues/485
Repeat the test for all style variations
